protostar/cheatable_starknet/cheatcodes/cairo_cheatcode.py

```diff
@@ -0,0 +1,13 @@
+from abc import ABC
```

Comment:
I think `CairoCheatcodes` should be in `cairo_testing`. `CairoCheatcodes` are basically views in the Cairo testing context.

Reply:
They have more in common with `CheatableState` than cairo testing itself, i'm not entirely convinced on this one.

Reply:
Also they can modify the state so they're not views per-se

Reply:
> I am not sure, cairo_testing is responsible for running test, and CairoCheatcodes are the exposed view layer of cheatable starknet, they have nothing to do with test execution environment

> They have more in common with CheatableState than cairo testing itself, i'm not entirely convinced on this one.
> Also they can modify the state so they're not views per-se

I'd say they are somewhere between `CairoTesting` and `CheatableStarknet`. They adapt input and output data from user to cheaters in this case, and cheaters modify the state. In the future, similar adapters will be needed by the Scarb plugin. Scarb plugin shouldn't depend on a component that includes CairoTesting's adapters.

It's not super important, though. Resolving.